InterfaceParser/model/enum.py

```diff
@@ -0,0 +1,40 @@
+"""Enumeration.
+
+"""
+from collections import OrderedDict
+
+from model.interface_item_base import InterfaceItemBase
+
+
+class Enum(InterfaceItemBase):
+    """Enumeration.
+
+    :param name: item name
+    :param description: list of string description elements
+    :param design_description: list of string design description elements
+    :param issues: list of issues
+    :param todos: list of string todo elements
+    :param platform: optional platform (string or None)
+    :param scope: optional scope: internal, partner or none (none by defaul, means public)
+    :param since:
+    :param until:
+    :param deprecated:
+    :param removed:
+    :param history:
+
+    :param internal_scope: optional internal scope
+    :param elements: dictionary of enumeration elements (instances of EnumElement class)
+
+    """
+
+    def __init__(self, name, description=None, design_description=None,
+                 issues=None, todos=None, platform=None, internal_scope=None,
+                 elements=None, scope=None, since=None, until=None,
+                 deprecated=None, removed=None, history=None):
+        super(Enum, self).__init__(
+            name, description=description, design_description=design_description,
+            issues=issues, todos=todos, platform=platform, scope=scope,
```

Comment:
Is it necessary to parse the todo tag at all?

Reply:
This code came from legacy sdl_core generator/parser. 
I'm not really aware is it necessary to parse the todo tag, but as it came from sdl_core most likely it used somewhere in sdl_core.